### PR TITLE
Docs release 1.11.0 csi-4729

### PR DESCRIPTION
**Pull request by rivkap**

### CODE REVIEW

File: docs/content/release_notes/known_issues.md [hunk 1]
```diff
@@ -12,14 +12,15 @@ The following severity levels apply to known issues:
...
 |**CSI-2157**|Service|In extremely rare cases, too many Fibre Channel worker node connections may result in a failure when the CSI driver attempts to attach a pod. As a result, the `Host for node: {0} was not found, ensure all host ports are configured on storage` error message may be found in the IBM block storage CSI driver controller logs. <br>**Workaround:** Ensure that all host ports are properly configured on the storage system. If the issue continues and the CSI driver can still not attach a pod, contact IBM Support.|
-|**CSI-4554**|Low|During HostDefinition creation no success event is emitted when the status is in a _Ready_ state.<br>**Workaround:** No workaround available.|
+|**CSI-4554**|Low|During HostDefinition creation, no success event is emitted when the status is in a _Ready_ state.<br>**Workaround:** No workaround available.|
```
Comment (matancarmeli7) [inline docs/content/release_notes/known_issues.md:25]: this one is solved

File: docs/content/release_notes/supported_storage.md [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 |--------------|-----------------|
-|FlashSystem™ A9000|12.3.2.c or later|
-|FlashSystem A9000R|12.3.2.c or later|
 |Spectrum Virtualize™ family including Spectrum Virtualize as software only, Spectrum Virtualize for Public Cloud, SAN Volume Controller (SVC), and FlashSystem® family members built with Spectrum® Virtualize (including FlashSystem 5xxx, 7xxx, 9xxx)|7.8.x, 8.2.x, 8.3.x, 8.4.x, 8.5.x|
 |DS8000® family|8.x and higher with same API interface|
 
 **Note:**
 
 - For the latest microcode storage support information, see the [Lifecycle and support matrix](https://www.ibm.com/docs/en/stg-block-csi-driver?topic=SSRQ8T/landing/csi_lifecycle_support_matrix.html).
+- Starting from CSI driver 1.11.0 release, FlashSystem A9000 and A9000R storage systems are no longer supported.
```
Comment (matancarmeli7) [inline docs/content/release_notes/supported_storage.md:13]: I talked to @erantzabari and we don't want to explicitly write that we don't support A9000, can you remove this line?

File: docs/content/release_notes/known_issues.md (hunk 1 above)
Comment (rivkap) [reply docs/content/release_notes/known_issues.md]: moved to fixed issues

File: docs/content/configuration/creating_volumereplication.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ When replicating a volume, be sure to follow all of the replication configuratio
 
     For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](creating_pvc.md).
 
-    **Note:** Use the `spec.csi.volumeHandle` of the relevant target PersistentVolume (PV) for the `replicationHandle` value.
+    **Important:** If you are not using policy-based replication, use the `spec.csi.volumeHandle` of the relevant target PersistentVolume (PV) for the `replicationHandle` value. If policy-based replication is being used, **do not** use the `replicationHandle` value. Using the `replicationHandle` while policy-based replication is defined in the VolumeReplicationClass results in the following error message: `got an invalid parameter: replicationHandle`.
```
Comment (ArbelNathan) [inline docs/content/configuration/creating_volumereplication.md:15]: we need to add description about configuring with the VolumeGroup Type for policy-based replication. maybe even new yaml

Comment (rivkap) [reply docs/content/configuration/creating_volumereplication.md]: updated FYI @ELENAGER  please review the changes

File: docs/content/release_notes/supported_storage.md (hunk 2 above)
Comment (rivkap) [reply docs/content/release_notes/supported_storage.md]: done

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Review by matancarmeli7 — APPROVED** [review]

File: docs/content/configuration/creating_volumereplication_pbr.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    kind: VolumeReplication
+    metadata:
+      name: demo-volumereplication
+      namespace: default
+    spec:
+      volumeReplicationClass: demo-volumereplicationclass
+      replicationState: primary
+      dataSource:
+        kind: VolumeGroup
+        name: demo-pvc-file-system  # Ensure that this is in the same namespace as VolumeReplication.
```
Comment (annaparienti) [inline docs/content/configuration/creating_volumereplication_pbr.md:26]: this should be the name of the volume group

File: docs/content/release_notes/changelog_1.11.0.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# 1.11.0 (January 2023)
+
+IBM® block storage CSI driver 1.11.0 added new support and enhancements.
+- Added dynamic host definition enhancements
+- New Spectrum Virtualize family system support for policy-based replication and dynamic volume groups
+- Additional orchestration platform support for Red Hat OpenShift 4.12 and Kubernetes 1.25
```
Comment (annaparienti) [inline docs/content/release_notes/changelog_1.11.0.md:6]: shouldn't it be 1.26?

File: docs/content/configuration/creating_volumereplication_pbr.md (hunk 4 above)
Comment (rivkap) [reply docs/content/configuration/creating_volumereplication_pbr.md:26]: @ELENAGER

**Pull request merged**